WORKSPACE

```diff
@@ -55,11 +55,13 @@ py_repositories()
 
 http_archive(
     name = "io_bazel_rules_closure",
-    sha256 = "6a900831c1eb8dbfc9d6879b5820fd614d4ea1db180eb5ff8aedcb75ee747c1f",
```

Comment:
Thanks, Yating!

One question... how do we get these values? Should we just comment them out? Or is it easy to return to this if we need to?

Reply:
Ok, but how about the urls? Where/how did you find this new value to reference? Is there a guide for how to update these references?

Is it just that these particular versions are the latest ones available at http://mirror.tensorflow.org?